compiler-rt/lib/sanitizer_common/sanitizer_platform_interceptors.h

```diff
@@ -191,7 +191,7 @@
 
 #define SANITIZER_INTERCEPT_PREADV \
   (SI_FREEBSD || SI_NETBSD || SI_LINUX_NOT_ANDROID)
-#define SANITIZER_INTERCEPT_PWRITEV SI_LINUX_NOT_ANDROID
+#define SANITIZER_INTERCEPT_PWRITEV (SI_FREEBSD || SI_LINUX_NOT_ANDROID)
```

Comment:
Fine if you don't know: is there a reason this isn't the same as SANITIZER_INTERCEPT_PREADV? I feel like a system would either support both or neither.

Reply:
Fair enough I can update :)